adabot/lib/circuitpython_library_validators.py

```diff
@@ -855,7 +855,17 @@ def gather_insights(self, repo, insights, since, show_closed_metric=False):
                         )
                         insights["merged_prs"].append(pr_link)
 
-                        insights["pr_merged_authors"].add(pr_info["user"]["login"])
+                        pr_author = pr_info["user"]["login"]
+                        if pr_author == "weblate":
+                            pr_commits = github.get(str(pr_info["url"]) + "/commits")
+                            if pr_commits.ok:
+                                for commit in pr_commits.json():
+                                    author = commit.get("author")
+                                    if author:
+                                        insights["pr_merged_authors"].add(author["login"])
```

Comment:
Do we want to guard against `weblate` being added here?

Possibly something like:
```python
if author and "weblate" not in author.get("login", "weblate"):
    insights["pr_merged_authors"].add(author["login"])
```

Reply:
I thought about that too, but left it out.  I can change it if you prefer.

Reply:
I have no super-duper overriding opinion to exclude it; was more just a question. We can leave it in for now. I mean, a shoutout to a helpful service isn't the worst thing ever. 😄